<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IDManager</class>
 <widget class="QWidget" name="IDManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="cbIP">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>此IP可作为别人搜索的关键词</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCopy">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>将IP复制到剪贴板</string>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leID">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>此ID可作为别人搜索的关键词</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>本机ID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbClose">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbRefresh">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>刷新本地IP</string>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
  <widget class="QLabel" name="laTip">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>91</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ID可用</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCopyID">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>将IP复制到剪贴板</string>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
